### PR TITLE
The beginning: Moves CC to Z1 and station to Z2

### PR DESCRIPTION
**Pull request by AffectedArc07**
## What Does This PR Do
This thus marks the beginning of map rotation. This moves CC to Z1, which is part of map rotation, and should be done in its own PR. A lot of stuff is hardcoded for Z1 and Z2, so this allows us to isolate errors outside of the main map rotation code.

## Why It's Good For The Game
Its time

## Changelog
:cl: AffectedArc07
experiment: The station is now Z2
experiment: CentComm is now Z1
/:cl:

### CODE REVIEW

**Comment by Kiyahitayika** [comment]
Rotate your maps. Rotate your maps. R-r-rotate your maps.
For Science!

File: _maps/cyberiad.dm [hunk 1]
```diff
@@ -3,20 +3,19 @@ All z-levels should be identical in size. Their numbers should not matter.
 The order of z-levels should not matter as long as their attributes are properly defined at MAP_TRANSITION_CONFIG.
 Old code checked for the number of the z-level (for example whether there are any revheads on Z1),
```
Comment (farie82) [inline _maps/cyberiad.dm:4]: Can most likely be removed since I can't find an instance where a numeric or define value is checked

Comment (AffectedArc07) [reply _maps/cyberiad.dm:4]: Why remove it? It says what the code used to do, and why you should use the new system.

Comment (farie82) [reply _maps/cyberiad.dm:4]: It would suggest that there is still old code which checks it

Comment (AffectedArc07) [reply _maps/cyberiad.dm:4]: `checked` is past tense, not `checks`.

**Comment by AffectedArc07** [comment]
I can do this in map standardisation which is actually happening soon, lol

**Pull request closed without merging**